tests/v2/actions/clustertemplates/clustertemplateconfig.go

```diff
@@ -0,0 +1,19 @@
+package clustertemplates
+
+import (
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+)
+
+type ClusterTemplateConfig struct {
+	RKE1TemplateID         string             `json:"clusterTemplateId" yaml:"clusterTemplateId"`
+	RKE1TemplateRevisionID string             `json:"clusterTemplateRevisionId" yaml:"clusterTemplateRevisionId"`
+	RKETemplateAnswers     *management.Answer `json:"answers,omitempty" yaml:"answers,omitempty"`
+}
+
+type ClusterTemplateRevisionConfig struct {
+	Name                  string                   `json:"name,omitempty" yaml:"name,omitempty"`
+	RKE1KubernetesVersion string                   `json:"rke1kubeversion,omitempty" yaml:"rke1kubeversion,omitempty"`
+	BackupConfig          *management.BackupConfig `json:"backupConfig,omitempty" yaml:"backupConfig,omitempty"`
+	NetworkPlugin         string                   `json:"plugin,omitempty" yaml:"plugin,omitempty"`
+	Questions             []management.Question    `json:"questions,omitempty" yaml:"questions,omitempty"`
+}
```

Comment:
typically we don't want to create new objects to represent data we already have structs for, when possible. 

Would it be possible for you to use pkg/apis/management.cattle.io/v3 -> ClusterTemplate and ClusterTemplateRevision? To me, it looks like the info you have in the new structs are contained directly in the Spec of the management objects